go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	github.com/BurntSushi/toml v0.4.1 // indirect
 	github.com/agext/levenshtein v1.2.2 // indirect
 	github.com/apparentlymart/go-textseg/v13 v13.0.0 // indirect
-	github.com/aws/aws-sdk-go v1.25.3 // indirect
```

Comment:
I think these changes might have been included accidentally. I don't see what causes these translative dependencies to need to be updated, but maybe I missed something.

Reply:
This comes from the `tools` package that imports other packages as a side-effect. It's odd that that package wasn't updated in previous commits.
```
% go mod why -m github.com/aws/aws-sdk-go
# github.com/aws/aws-sdk-go
csbbrokerpakgcp/tools
github.com/cloudfoundry/cloud-service-broker
github.com/cloudfoundry/cloud-service-broker/cmd
github.com/cloudfoundry/cloud-service-broker/pkg/brokerpak
github.com/cloudfoundry/cloud-service-broker/internal/brokerpak/packer
github.com/hashicorp/go-getter
github.com/aws/aws-sdk-go/aws
```

Reply:
I'll try re-running `go mod tidy` just in case.

Reply:
Re-running `go mod tidy` did not yield any changes